packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -244,41 +237,30 @@ const FooterArea = styled.div`
 `
 
 const FormData = styled.div`
+  padding-top: 24px;
   background: ${({ theme }) => theme.colors.white};
   color: ${({ theme }) => theme.colors.copy};
-  padding: 32px;
-  border-radius: 4px;
-  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
-    padding: 24px;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
```

Comment:
I think this was supposed to change the breakpoint from 'lg' to 'md'. But the css rules inside got removed accidentally maybe?